ProcessMaker/processmaker
# Implement Preview Template Feature in Create New Screen Modal

This PR implements the 'Preview Template' functionality within the Create New Screen modal. Selecting 'Preview Template' displays an image viewer for the thumbnails uploaded to a template. Additionally, users can preview the template's custom CSS code.

## Solution

- Added frontend components and UI elements to support template preview functionality.
- Integrated image viewer for thumbnail previews.
- Implemented logic to display custom CSS code preview within the modal.

## How to Test

1. Open the Create New Screen modal.
2. Upload thumbnails for the template.
3. Click on the 'Preview Template' option.
4. Verify that the uploaded thumbnails are displayed in the image viewer.
5. Check if the custom CSS code is properly rendered in the preview section.
6. Optionally, make changes to the thumbnails or CSS code and observe the live preview.

## Related Tickets & Packages

[FOUR-13496](https://processmaker.atlassian.net/browse/FOUR-13496)

ci:next
ci:deploy

## Code Review Checklist


[FOUR-13496]: https://processmaker.atlassian.net/browse/FOUR-13496?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
